hack/graph-util.py

```diff
@@ -30,6 +30,13 @@
 _LOGGER = logging.getLogger(__name__)
 _LOGGER.setLevel(logging.DEBUG)
 
+def get_architecture(meta):
+    return meta['image-config-data']['architecture']
+
+def get_log_version(node):
+    meta = node['meta']
+    arch = get_architecture(meta)
+    return "%s+%s" % (node['version'], arch)
```

Comment:
You probably want:

```python
semver_match = _VERSION_REGEXP.match(node['version'])
if not semver_match:
    raise ValueError('version {} is not a valid Semantic Version'.format(node['version']))
return '{}+{}'.format(remove_build(semver_match=semver_match), arch)
```

to guard against doubling the build metadata.  Although we have not actually cut a release with build metadata (yet).  I'm also fine with rendering this as `{} ({}).format(node['version'], arch)`, and not trying to cram it into SemVer.

Reply:
I don't care how we do it, just so long as when we look at the output we know what arch's changes are being applied to.  So I just took your suggestion.